services/slam/server.go

```diff
@@ -143,6 +144,23 @@ func (server *subtypeServer) GetInternalState(ctx context.Context, req *pb.GetIn
 	}, nil
 }
 
+// GetPointCloudMapStream calls the callback function returned from the request to the slam service's
+// GetPointCloudMapStream and sends the chunk of the current pointcloud map to the provided stream.
+func (server *subtypeServer) GetPointCloudMapStream(req *pb.GetPointCloudMapStreamRequest,
+	stream pb.SLAMService_GetPointCloudMapStreamServer,
+) error {
+	return errors.New("unimplemented stub")
+}
+
+// GetInternalStateStream calls the callback function returned from the request to the slam service's
```

Comment:
Given that this is a GRPC server method I don't we should only document what this does from the caller's perspective, not how it does it. 

Recommended change:
// GetInternalStateStream returns the internal state of the slam service's slam algo in a stream of byte chunks.